asset/asset.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "file-assets",
-    "version": "0.8.3",
+    "version": "0.8.4",
```

Comment:
This looks like a feature was added, we should make it a minor release `0.9.0`

Reply:
Fixed